PowerShell/ScubaGear/Modules/Support/Support.psm1

```diff
@@ -849,6 +863,46 @@ function New-Config {
 
     $PartnerDomainImpersonationProtectionNamespace = "MS.DEFENDER.2.3v1"
 
+    $OmissionNamespace = "OmitPolicy"
+
+    # List to track which policies the user specified in $OmitPolicies are properly formatted
+    $OmitPolicyValidated = @()
+
+    # Hashmap to structure the ignored policies template
+    $config[$OmissionNamespace] = @{}
+
+    foreach ($Policy in $OmitPolicy) {
```

Comment:
This logic validates the formatting of $OmitPolicies when a user first invokes New-Config, but I'm not seeing any validation in cases where the config.yml is modified after creation. If I incorrectly set a key in `OmitPolicy` to `MSAAD4.1v1` or `MS.GWS.AAD4.2v1` these omissions are skipped and the policy runs normally. 

Perhaps an additional unit test to verify malformed policy IDs are excluded from CreateReport.psm1 and the policy is expected to run by default?

Reply:
Done. I also modified the load config function to output warnings for invalid IDs. Tagging @tkol2022 for awareness.
![image](https://github.com/user-attachments/assets/59ed19ed-99c4-4d5d-8575-4aed1f72fcd1)